README.md

```diff
@@ -1,2 +1,27 @@
 # Create database users
-Once the consignment api database is created, we need to create users for the API and the migration task to use. This means that we don't have to use the master username and password to access the database, we can control permissions using GRANT and we can use IAM authentication for the database. 
\ No newline at end of file
+Once the consignment api database is created, we need to create users for the API and the migration task to use. This means that we don't have to use the master username and password to access the database, we can control permissions using GRANT and we can use IAM authentication for the database. 
+
+## Running the lambda in AWS
+As this is something that we only need to run once after the database is created, we just need to run the lambda manually.
+
+`aws lambda invoke --function-name tdr-create-db-users-$STAGE`
+
+## Running the lambda locally.
+If you want to run this against a local database, there are a few extra steps.
+
+Create the database if it doesn't exist. It will need the correct username and password. You can use docker
+
+`docker run --name postgres-db-users -e POSTGRES_PASSWORD=password -e POSTGRES_USER=tdr -e POSTGRES_DB=consignmentapi -d -p 5432:5432 postgres`
```

Comment:
The name of the container in the command is still: `postgres`

Reply:
`postgres` is the name of the image. the `--name` argument is the one that gives the container its name.

Reply:
Fair point, doh!